docs/source/examples/hyperband.ipynb

```diff
@@ -0,0 +1,244 @@
+{
```

Comment:
Long-term, this ideally goes in `dask-examples`, and we'll pull it in as part of the doc build automatically.

For now, where should it go? Should we include it as part of this PR, or remove it for now? If it goes in dask examples right now, we would have to pip install this branch till there's a dask-ml release with hyperband.

Reply:
I need to update this notebook; I haven't touched it for a while.

What the release schedule/policy for Dask-ML?

Reply:
As needed, so about 1 minute after merging hyperband most likely :)

Releases are fairly cheap to do now.